src/models/model_storage.rb

```diff
@@ -84,4 +80,30 @@ def bottle_specifications
     end
     specifications
   end
+
+  def possible_attachable_user_names
+    return @possible_names unless @possible_names.nil?
+
+    @possible_names = []
+    puts 'Loaded files for attachable users:'
+    Dir.glob(ProjectHelper.component_directory + '/attachable_users/*.skp') do |model_file_path|
+      # File scheme is '.../number-name-short_name-weight_in_grams.skp'
+      # TODO: Read in the weight
```

Comment:
please add a sample file name in this comment

Reply:
and why are `child.skp` and `boy.skp` not following this scheme? Is this part of the todo comment?

Reply:
I now did the todo, so now they follow :D 
The comment was unfortunatly also just a copy paste, now I made it fit